argilla-io/argilla
# feat: #4118 Added `ResponseStatusFilter` enum in `__init__`

# Description

This PR eases the usage of `ResponseStatusFilter`.  No additional dependencies are introduced with this change.

```
import argilla as rg
...
dataset.filter_by(response_status=rg.ResponseStatusFilter.submitted)

```

Closes #4118 

**Type of change**


**How Has This Been Tested**

To verify this Feature:

- Clone the repository.
- Execute code snippet involving `import argilla as rg`
- Execute the modified code snippet involving `rg.ResponseStatusFilter`.
- Confirm that the intended behavior aligns with the documented usage (https://docs.argilla.io/en/latest/reference/python/python_client.html#argilla.client.feedback.dataset.remote.dataset.RemoteFeedbackDataset.filter_by) .

**Checklist**

- [ ] I added relevant documentation
- [x] I followed the style guidelines of this project
- [x] I did a self-review of my code
- [ ] I made corresponding changes to the documentation
- [x] I filled out [the contributor form](https://tally.so/r/n9XrxK) (see text above)
